src/Sulu/Bundle/WebsiteBundle/Tests/Unit/Twig/ContentTwigExtensionTest.php

```diff
@@ -158,7 +158,7 @@ protected function setUp(): void
             $this->logger->reveal(),
             $this->securityChecker->reveal(),
             $this->webspaceManager->reveal(),
-            null,
+            $this->prophesize(RequestStack::class)->reveal(),
```

Comment:
better directlydo `new RequestStack()` no need to mock such objects

Reply:
If we're not mocking the object though, we can not assert that calls on it have been made. Is that okay?

Reply:
We can test all what is important. Put different request objects into it to test different requests: `$this->requestStack->push($request)` and we can test state of current request after method was called `$this->requestStack->getCurrentRequest()`